solutions/ac-GCDOnBlackboard.mdx

```diff
@@ -76,4 +76,45 @@ int main(){
 
 ```
 </CPPSection>
+	
+<JavaSection>
+
+```java
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+        BufferedReader input = new BufferedReader(new InputStreamReader(System.in));
```

Comment:
change indents from spaces to tabs :)

Reply:
use: https://usaco.guide/editor/ and highlight all, command palette, and search up tabs